config/environment.js

```diff
@@ -16,31 +16,26 @@ module.exports = function (environment) {
         Date: false,
       },
     },
-    environmentName: "{{ENVIRONMENT_NAME}}",
+    environmentName: '{{ENVIRONMENT_NAME}}',
     fastboot: {
-      hostWhitelist: [/^localhost:\d+$/,
-                      "localhost",
-                      "publicatie.gelinkt-notuleren.vlaanderen.be",
-                      "publicatie.dev.gelinkt-notuleren.lblod.info",
-                      "publicatie.gebruikerssessie.gelinkt-notuleren.lblod.info",
-                      "publicatie.gelinkt-notuleren.lblod.info",
-                      "backend" //mu-semtech
-                     ]
+      hostWhitelist: [
+        /^localhost:\d+$/,
```

Comment:
This is going to give issues if `@embroider/macros` is part of the node_modules tree since it expects that the env config is serializable.
```suggestion
        /^localhost:\d+$/.toString(),
```